rosbridge_library/src/rosbridge_library/protocol.py

```diff
@@ -32,10 +32,7 @@
 
 import rospy
 import time
-from rosbridge_library.internal.exceptions import InvalidArgumentException
```

Comment:
These imports are important - they hoist the exceptions from the `internal.exceptions` namespace to `rosbridge_library.protocol` (see https://travis-ci.org/RobotWebTools/rosbridge_suite/jobs/329044753)

Reply:
mmm ... i thought they were placed in the tests by mistake or so ... i'll return them in the morning then 👍

Reply:
Just out of curiosity, why do we need to do that? Why not just import them from internal.exceptions?

Reply:
Well, I'm not sure we absolutely need to, but removing those imports now would be a breaking API change in any case, which means a major version bump.

Reply:
i think they're imported from protocol only in the tests ... i did a quick search in the repo, and it seems only the tests use them like that.

Reply:
In our code, yes. I'm talking about external code that uses `rosbridge_library` to implement some sort of custom bridge. There's not going to be a lot of that, but as I said, we'd be changing our public API. Let's keep the imports for now, and make a note somewhere to keep them in mind for when there's more reasons to justify a major version bump :)

Reply:
yes ... i returned them for now. If we need to remove them later then it should be in another PR anyways. 
Thanks for the explanation :)